external/storm-kafka/README.md

```diff
@@ -86,7 +93,7 @@ The KafkaConfig class also has bunch of public variables that controls your appl
     public int fetchMaxWait = 10000;
     public int bufferSizeBytes = 1024 * 1024;
     public MultiScheme scheme = new RawMultiScheme();
-    public boolean ignoreZkOffsets = false;
+    public boolean ignoreStoredOffsets = false;
```

Comment:
is there a mechanism/section in the release notes to call out this change?  changing this var name would break existing code.